cmd/podman/trust.go

```diff
@@ -127,23 +131,21 @@ func showTrustCmd(c *cli.Context) error {
 		return nil
 	}
 
-	var policyContentStruct trust.PolicyContent
-	if err := json.Unmarshal(policyContent, &policyContentStruct); err != nil {
-		return errors.Errorf("could not read trust policies")
+	policyContentStruct, err := trust.GetPolicy(policyPath)
+	if err != nil {
+		return errors.Wrapf(err, "could not read trust policies")
 	}
-	policyJSON, showOutputMap, err := trust.GetPolicy(policyContentStruct, systemRegistriesDirPath)
+	policyJSON, showOutputMap, err := getPolicyOutput(policyContentStruct, systemRegistriesDirPath)
 	if err != nil {
-		return errors.Wrapf(err, "error reading registry config file")
+		return errors.Wrapf(err, "error formating trust policy")
 	}
 	if c.Bool("json") {
-		var outjson interface{}
 		outjson = policyJSON
 		out := formats.JSONStruct{Output: outjson}
 		return formats.Writer(out).Out()
 	}
 
-	sortedRepos := sortPolicyJSONKey(policyJSON)
-	var output []interface{}
+	sortedRepos := sortShowOutputMapKey(showOutputMap)
```

Comment:
(Non-blocking: in a sense, this sorting and computation of `output` is also part of the non-JSON formatting and could happen inside `getPolicyOutput` or so.)